index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Apple clone</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/e6d0becbe7.js"></script>
+</head>
+
+<body>
+    <main>
+        <section class="hero">
+            <section class="menu">
+                <ul class="menu-items">
+                    <li class="menu-item"><img src="assets/images/apple-icon.png" alt="apple-icon"></li>
+                    <li class="menu-item">Store</li>
+                    <li class="menu-item">Mac</li>
+                    <li class="menu-item">iPod</li>
+                    <li class="menu-item">iPhone</li>
+                    <li class="menu-item">iPad</li>
+                    <li class="menu-item">iTunes</li>
+                    <li class="menu-item">Support</li>
+                        
+                        <li class="menu-item"><input type="text" /><i class="fas fa-search"></i></li>
+
+            
+
+                </ul>
+            </section>
+            <img src="assets/images/hero-img1.png" alt="what will your verse be" class="hero-text-1" />
+
+            <img src="assets/images/hero-p2.png" alt="everyone has something to share." class="hero-text-2" />
```

Comment:
These text images are also clickable links, let's contain it inside a `div` to be more modular and add the links around them.